ports/geckolib/glue.rs

```diff
@@ -707,8 +707,12 @@ pub extern "C" fn Servo_AnimationValue_Serialize(value: RawServoAnimationValueBo
     let uncomputed_value = AnimationValue::as_arc(&value).uncompute();
     let mut string = String::new();
     let rv = PropertyDeclarationBlock::with_one(uncomputed_value, Importance::Normal)
-        .single_value_to_css(&get_property_id_from_nscsspropertyid!(property, ()), &mut string,
-                             None, None /* No extra custom properties */);
+        .single_value_to_css(
+            &get_property_id_from_nscsspropertyid!(property, ()),
+            &mut CssWriter::new(&mut string),
```

Comment:
We should make this pass `buffer` directly. Mind filing an issue? Though I don't recall if @bholley's patch fixed this.